scu-es/src/fiskaltrust.Middleware.SCU.ES.VeriFactu/Models/MiddlewareState.cs

```diff
@@ -1,43 +1,47 @@
-
 using System.Text.Json;
 using System.Text.Json.Serialization;
 using fiskaltrust.ifPOS.v2;
 
 namespace fiskaltrust.Middleware.SCU.ES.Models;
 
-public class MiddlewareState
+public class MiddlewareStateData
 {
     [JsonPropertyName("ES")]
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
-    public MiddlewareQueueESState? ES { get; set; }
+    public MiddlewareStateDataES? ES { get; set; }
+
+    [JsonPropertyName("ftPreviousReceiptReference")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.Never)]
+    public List<Receipt>? PreviousReceiptReference { get; set; } = new();
```

Comment:
ftPreviousReceiptReference

Reply:
it's actually called that in the `JsonPropertyName` 🤔 I just did not want to have a camelcase name or a name starting with `Ft`in code so I removed the prefix 🤷